<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InternDiffView</class>
 <widget class="QMainWindow" name="InternDiffView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>InternDiffView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(32, 33, 72);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical { 
                                                border: none;
                                                background: white;
                                                width: 9.81px;
                                            }
QScrollBar::handle:vertical { 
                                                background-color:  rgba(1, 22, 51, 0.5);
                                                min-height: 273.41px;
														border: 0px solid red; 
														border-radius:4.905px;
														
                                            }
QScrollBar::add-line:vertical {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:vertical { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

QScrollBar:horizontal { 
                                                border: none;
                                                background: white;
                                             		
                                            }
QScrollBar::handle:horizontal { 
                                                background-color:  rgba(1, 22, 51, 0.5);
														border: 0px solid red; 
														border-radius:4.905px;
                                            }
QScrollBar::add-line:horizontal {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:horizontal { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="prevButton">
        <property name="styleSheet">
         <string notr="true">border: 0px; </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="AppResources.qrc">
          <normaloff>:/Images/Resources/Previous.png</normaloff>:/Images/Resources/Previous.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;p&gt;&lt;b&gt;1. OCR Image&lt;b&gt;&lt;/p&gt;
&lt;p&gt; &lt;/p&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#graphicsView{
background-color:white; color:black;
}                
QScrollBar:vertical { 
                                                border: none;
                                                background: white;
                                                width: 9.81px;
                                            }
QScrollBar::handle:vertical { 
                                                background-color:  rgba(1, 22, 51, 0.5);
                                                min-height: 273.41px;
														border: 0px solid red; 
														border-radius:4.905px;
														
                                            }
QScrollBar::add-line:vertical {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:vertical { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

QScrollBar:horizontal { 
                                                border: none;
                                                background: white;
                                             		
                                            }
QScrollBar::handle:horizontal { 
                                                background-color:  rgba(1, 22, 51, 0.5);
														border: 0px solid red; 
														border-radius:4.905px;
                                            }
QScrollBar::add-line:horizontal {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:horizontal { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

</string>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::ScrollHandDrag</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="OCRtextLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;p&gt;&lt;b&gt;2. Initial Text&lt;b&gt;&lt;/p&gt;
&lt;p&gt; &lt;/p&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="ocroutput">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#ocroutput{
background-color:white; color:black;
}
QScrollBar:vertical { 
                                                border: none;
                                                background: white;
                                                width: 9.81px;
                                            }
QScrollBar::handle:vertical { 
                                                background-color:  rgba(1, 22, 51, 0.5);
                                                min-height: 273.41px;
														border: 0px solid red; 
														border-radius:4.905px;
														
                                            }
QScrollBar::add-line:vertical {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:vertical { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

QScrollBar:horizontal { 
                                                border: none;
                                                background: white;
                                             		
                                            }
QScrollBar::handle:horizontal { 
                                                background-color:  rgba(1, 22, 51, 0.5);
														border: 0px solid red; 
														border-radius:4.905px;
                                            }
QScrollBar::add-line:horizontal {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:horizontal { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Shobhika'; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="InternLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;p&gt;&lt;b&gt;3. Corrector's Output Text&lt;/b&gt;&lt;/p&gt;Changes made by Corrector: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="current">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#current{
background-color:white; color:black;
}
QScrollBar:vertical { 
                                                border: none;
                                                background: white;
                                                width: 9.81px;
                                            }
QScrollBar::handle:vertical { 
                                                background-color:  rgba(1, 22, 51, 0.5);
                                                min-height: 273.41px;
														border: 0px solid red; 
														border-radius:4.905px;
														
                                            }
QScrollBar::add-line:vertical {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:vertical { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

QScrollBar:horizontal { 
                                                border: none;
                                                background: white;
                                             		
                                            }
QScrollBar::handle:horizontal { 
                                                background-color:  rgba(1, 22, 51, 0.5);
														border: 0px solid red; 
														border-radius:4.905px;
                                            }
QScrollBar::add-line:horizontal {       
                            height: 0px;
                            subcontrol-position: bottom; 
                            subcontrol-origin: margin; 
								
                        } 
QScrollBar::sub-line:horizontal { 
                            height: 0 px; 
                            subcontrol-position: top; 
                            subcontrol-origin: margin; 
                        }

</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Shobhika'; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="NextButton">
        <property name="styleSheet">
         <string notr="true">border: 0px; </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="AppResources.qrc">
          <normaloff>:/Images/Resources/next.png</normaloff>:/Images/Resources/next.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>264</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>264</width>
          <height>37</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
	background: white;
	padding-top:15px;
	margin-top:-15px; 
   height:37.9px;
   font-size: 11pt !important;
}
</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="zoom_Out_Button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">#zoom_Out_Button {
	color: #202148;
	background: white;
	font-size: 24px;
	padding-right: 12px;
	padding-left: 12px;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider {
	background: white;
}

QSlider::groove:horizontal {
border: none;
background: rgba(1, 22, 51, 0.72);
height: 3.47px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: #202148;
border-radius: 1.7px;
width:  10px;
height: 26px;
margin-top: -10px;
margin-bottom: -10px;
}

QSlider::sub-page:horizontal:disabled {
background: grey;
}

QSlider::add-page:horizontal:disabled {
background: grey;
}

QSlider::handle:horizontal:disabled {
background: grey;
}
</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>0</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoom_In_Button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">#zoom_In_Button {
	color: #202148;
	background: white;
	font-size: 24px;
	padding-right: 12px;
	padding-left: 12px;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="zoom_level_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">#zoom_level_value {
	color: #202148;
	background: white;
}</string>
           </property>
           <property name="text">
            <string>100%</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppResources.qrc"/>
  <include location="AppResources.qrc"/>
  <include location="AppResources.qrc"/>
 </resources>
 <connections/>
</ui>
